pkg/cmd/init.go

```diff
@@ -157,6 +157,19 @@ func initCerts(cfg *config.MicroshiftConfig) ([]byte, *cryptomaterial.Certificat
 		).WithSubCAs(
 			cryptomaterial.NewCertificateSigner("kube-csr-signer", cryptomaterial.CSRSignerCertDir(certsDir), cryptomaterial.ClientCAValidityDays),
 		),
+		cryptomaterial.NewCertificateSigner(
+			"aggregator-signer",
+			cryptomaterial.AggregatorSignerDir(certsDir),
+			cryptomaterial.ClientCAValidityDays,
+		).WithClientCertificates(
+			&cryptomaterial.ClientCertificateSigningRequestInfo{
+				CertificateSigningRequestInfo: cryptomaterial.CertificateSigningRequestInfo{
+					Name:         "aggregator-client",
+					ValidityDays: cryptomaterial.ClientCertValidityDays,
+				},
+				UserInfo: &user.DefaultInfo{Name: "system:openshift-aggregator"},
+			},
+		),
```

Comment:
Should we remove this? https://github.com/openshift/microshift/pull/951/files#diff-a53aedd081647bc5a1bc5c1a969ad049f4ff08c9b4c5ba9408ea647488849234L219-L223

Reply:
absolutely